### ISSUE TITLE
v3.5/4.3: Create an experimental version of Umalqura calendar

### ISSUE THREAD

**Issue by MenoData** [MenoData/Time4J#313]
This issue is part of work on milestone M6 and is mainly about the Umalqura-data and not about a complete Hijri calendar. Once the calendar is finished together with some others then the milestone M6 can be closed.

**Comment by MenoData** [comment]
This is one single variant of Islamic (Hijri) calendar, see issue #91

**Comment by MenoData** [comment]
Postponed to v3.5 in order to release v3.4 sooner.

**Comment by MenoData** [comment]
The versioned serialization format will be postponed to another release (probably v3.6). Meanwhile the current implementation will be declared as experimental (because the serialization format will change). What is left now for release v3.5/4.3: Format support via a `CalendarFamily`.

**Comment by MenoData** [comment]
The current state is now good enough for a first release. More details in later releases.

**Comment by MenoData** [comment]
Closing was too early. Some operators (`incremented()` and `decemented()`) on the elements need to be overwritten (otherwise an exception happens). Furthermore, the era element should not be adjustable at all. Maybe there should also be support for `EpochDays`.
